arch/src/x86_64/mod.rs

```diff
@@ -718,6 +721,37 @@ pub fn generate_common_cpuid(
         });
     }
 
+    // mwait
+    // Intel-defined CPU features, CPUID level 0x00000001 (ECX)
+    // SAFETY: leaf are guaranteed to be non-null.
+    if unsafe { std::arch::x86_64::__cpuid(0x1) }.ecx & 1 << X86_FEATURE_MWAIT
+        == 1 << X86_FEATURE_MWAIT
+    {
+        // Thermal and Power Management Leaf, CPUID level 0x00000006 (EAX)
+        // SAFETY: cpuid called with valid leaves
+        if unsafe { x86_64::__cpuid(0x6) }.eax & 1 << X86_FEATURE_ARAT == 1 << X86_FEATURE_ARAT {
+            // Update some existing CPUID for X86_FEATURE_MWAIT
+            for entry in cpuid.as_mut_slice().iter_mut() {
+                if entry.function == 0x1 {
+                    // set mwait bit
+                    if entry.ecx & 1 << X86_FEATURE_MWAIT != 1 << X86_FEATURE_MWAIT {
+                        info!("mwait function enable.");
```

Comment:
Nit: should be `info!("mwait instruction cpuid leaf enabled.")`

Reply:
Yes, it is good suggestion.